samples/nginx/deployment.yaml

```diff
@@ -2,8 +2,13 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: nginx
+  annotations:
```

Comment:
I don't think we need to touch `samples` for this PR. Because the samples are used by `quick start` we should keep it simple. We can document the newly introduced overriders somewhere.

Reply:
ok